sdk/src/wallet/operations/transaction/submit_transaction.rs

```diff
@@ -22,11 +22,14 @@ where
     ) -> crate::wallet::Result<BlockId> {
         log::debug!("[TRANSACTION] submit_signed_transaction");
 
+        let block_id = self
+            .submit_basic_block(Some(Payload::from(payload)), issuer_id, true)
+            .await?;
+
         #[cfg(feature = "events")]
         self.emit(WalletEvent::TransactionProgress(TransactionProgressEvent::Broadcasting))
             .await;
 
```

Comment:
Still not sure why the order should be changed `Broadcasting` is not after it was broadcasted

Reply:
Then `Broadcasting` doesn't make sense. Because it's being emitted even before the block has been signed, which might take a while since you could be signing with ledger, perhaps I should rename it to `Broadcasted` ? What do you think?

Reply:
Or move it after signing?

Reply:
Sure, that would do too

Reply:
Done @Thoralf-M